### ISSUE TITLE
Appears to be broken on Ember 3.x

### ISSUE THREAD

**Issue by jacobq** [BryanHunt/ember-split-view#47]
```
loader.js:247 Uncaught Error: Could not find module `ember` imported from `ember-resize/services/resize`
    at missingModule (loader.js:247)
    at findModule (loader.js:258)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
    at requireModule (loader.js:24)
    at resolveInitializer (index.js:34)
    at registerInitializers (index.js:47)
    at exports.default (index.js:28)
...
```

```
Uncaught TypeError: Cannot read property 'listeners' of undefined
    at window.onerror 
...
```

**Comment by BryanHunt** [comment]
I'm swamped right now and probably can't look at this till the Christmas holidays.  If anyone wants to take a shot at a PR, I'll give it some priority.

**Comment by hmuraj** [comment]
Try manually updating ember-resize 
https://www.npmjs.com/package/ember-resize

0.3.0 works

**Comment by BryanHunt** [comment]
This should be fixed with v1.2.0
